lib/tests/ruby/ruby.cc

```diff
@@ -528,7 +528,7 @@ SCENARIO("custom facts written in Ruby") {
         }
     }
     GIVEN("a custom on_message block") {
-        log_capture capture(level::debug);
+        log_capture capture(level::warning);
```

Comment:
I can do that. Should I use another way of checking if the file is there though, to avoid that in general? I just wanted to make use of our existing utility functions, but could be it's not a good use case for that method.